app/views/index.html

```diff
@@ -0,0 +1 @@
+Hello world
```

Comment:
Is this view something that we need to keep now that its working? I thought this was just added to help with the dev process but I could be misremembering that conversation.